src/store/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from "redux";
```

Comment:
Consider going through the app an separate (group) different levels of imports to improve readability.